Dockerfile

```diff
@@ -86,6 +90,9 @@ RUN git clone --single-branch --branch ${ONNXRUNTIME_BRANCH} --recursive ${ONNXR
 
 ADD tools/build_and_test_onnxrt.sh /onnxruntime/build_and_test_onnxrt.sh
 
+
+RUN cget -p /usr/local install kitware/cmake@v3.20.0
```

Comment:
It would be best to make these docker file changes in a seperate PR though.

Reply:
I think I'll wait for the newest one to come out. Seeing make check failures when running in this 5.2.x container vs the stock one, whereas when I run make check in the stock 5.0.x everything makes sense and is there.

The only thing I could see requiring a change is the sha-1 pointing to onnxruntime to point to the latest Execution Provider that leverages these changes